artman/utils/task_utils.py

```diff
@@ -20,26 +20,26 @@
 import six
 
 
-def get_gradle_task_output(task_name, task_path):
-    task_path = os.path.realpath(os.path.expanduser(task_path))
-    output = subprocess.check_output(['./gradlew', task_name], cwd=task_path)
-    if isinstance(output, six.binary_type):
-        output = output.decode('ascii')
+def get_java_tool_path(toolkit_path, tool_name):
+    path = os.path.join(toolkit_path, 'build/toolpaths/{}'.format(tool_name))
```

Comment:
`os.path.join(toolkit_path, 'build', 'toolpaths', tool_name)`

Reply:
That makes so much more sense.